Dockerfile

```diff
@@ -12,10 +12,15 @@ RUN apt-get update \
         ninja-build jq \
   && rm -rf /var/lib/apt/lists/*
 
-ADD docs/requirements-doc.txt requirements.txt /
+# Normally, it is a bad idea to install rustup and cargo in
+# system directories (it should not be shared between users),
+# but this docker image is only for building packages, so I hope it is ok.
+RUN wget -q -O - https://sh.rustup.rs | \
+    RUSTUP_HOME=/usr CARGO_HOME=/usr sh -s -- -y --profile minimal --no-modify-path
 
-RUN pip3 --no-cache-dir install -r /requirements.txt \
-  && pip3 --no-cache-dir install -r /requirements-doc.txt
```

Comment:
Installing requirements-doc.txt fails because it now contains [a relative path to pyodide-build](https://github.com/pyodide/pyodide/blob/fd49100981dd09ec1c00238456baca58d513bf4c/docs/requirements-doc.txt#L18). I think we don't need build docs inside the docker image anyway.

Reply:
> we don't need build docs inside the docker image anyway.

I think the docker image should be able to build the docs. Why not `cd src && pip install -r docs/requirements-docs`.

Reply:
> I think the docker image should be able to build the docs

Okay, I reverted the change.

> Why not cd src && pip install -r docs/requirements-docs.

Actually, `pyodide-build` directory needs to be copied into the docker image.